src/igvfd/tests/test_types_sample.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+
+def test_file_sets_link(testapp, tissue, measurement_set, analysis_set_base, curated_set_genome):
+    testapp.patch_json(
+        measurement_set['@id'],
+        {
+            'samples': [tissue['@id']]
+        }
+    )
+    res = testapp.get(tissue['@id'])
+    assert res.json.get('file_sets') == [measurement_set['@id']]
+    testapp.patch_json(
+        analysis_set_base['@id'],
+        {
+            'samples': [tissue['@id']]
+        }
+    )
+    testapp.patch_json(
+        curated_set_genome['@id'],
+        {
+            'samples': [tissue['@id']]
+        }
+    )
+    res = testapp.get(tissue['@id'])
+    assert set(res.json.get('file_sets')) == set(
```

Comment:
the right side of the equation should be `set(measurment_set['id'],,,,....`. there is no need for [] inside the set()

Reply:
this change throws and error:
>       assert set(res.json.get('file_sets')) == set(measurement_set['@id'], analysis_set_base['@id'], curated_set_genome['@id'])
E       TypeError: set expected at most 1 argument, got 3

set() is used for type casting, I don't think it defines a set, { } can define a set.